api/src/main/java/org/pmiops/workbench/api/ConfigController.java

```diff
@@ -34,7 +34,7 @@
             .unsafeAllowSelfBypass(config.access.unsafeAllowSelfBypass)
             .enableNewAccountCreation(config.featureFlags.enableNewAccountCreation)
             .requireInvitationKey(config.access.requireInvitationKey)
-            .requireInstitutionalVerification(
-                config.featureFlags.requireInstitutionalVerification));
+            .requireInstitutionalVerification(config.featureFlags.requireInstitutionalVerification)
+            .enableCBAgeTypeOptions(config.featureFlags.enableCBAgeTypeOptions));
```

Comment:
Why do we need this?

Reply:
Per [this comment](https://github.com/all-of-us/workbench/blob/master/ui/src/environments/environment-type.ts#L59), that is now the preferred way to add feature flags.